rules/xcodeproj.bzl

```diff
@@ -701,9 +701,12 @@ def _populate_xcodeproj_targets_and_schemes(ctx, targets, src_dot_dots, all_tran
         target_settings["SWIFT_ACTIVE_COMPILATION_CONDITIONS"] = " ".join(
             ["\"%s\"" % d for d in defines_without_equal_sign],
         )
-        target_settings["BAZEL_LLDB_SWIFT_EXTRA_CLANG_FLAGS"] = " ".join(
-            ["-D%s" % d for d in target_info.cc_defines.to_list()],
-        )
+        extra_clang_flags = ["-D%s" % d for d in target_info.cc_defines.to_list()]
+
+        if virtualize_frameworks:
+            extra_clang_flags += ["-Xcc %s" % d for d in _objc_copts_for_target(target_name, all_transitive_targets)]
```

Comment:
I assume this list can potentially be very long? Any chance of hitting ARGMAX somewhere? Just mentioning this in case it's worth running this against one complex app first

Ideally that would be automated but we haven't had the time yet to improve CI and run it against one of the internal apps

Reply:
looking at the implementation of `_objc_copts_for_target` I don't think the list will be long since it is for `target_name` only not all of its deps?

Reply:
Right yeah, also the same list goes in `OTHER_CFLAGS` so what I'm bringing up here would've been noticed before if it was a problem. K thx!

Reply:
Yeah I don't think either of these would be long in practice, the `ARGMAX` issue can probably happen without virtual frameworks enabled given a large enough app